### DISCUSSION TITLE
Style editing: Clearer differentiation between "asserted" and "keep as-is" values

### DISCUSSION THREAD

**Discussion by fkoglin** [freeplane/freeplane#1512] [Ideas]
1. Any non-asserted values in styles should be clearly marked as such, i.e. as "unchanged" or "keep as-is" or whatever, instead of showing the values for the Default style. 

2. The button label for "Revert" should be expanded to "Revert to Default" (or "Revert to keep as-is" if 1 is implemented).

**Is your feature request related to a problem? Please describe.**
Right now, when I create a new style from scratch, it’s pre-filled with the values taken from the "Default" style. However, they’re not actually "asserted", i.e. if I apply the newly created style to a node with automatic level styles, **nothing changes**. This is fairly unintuitive to me: If there’s a value visible in a style, I’d expect it to be applied.

There’s also **only a tiny visual difference** when you change the default value from "keep as-is" to "overwrite": The tiny "Revert" button to the right. Which also confused me, because I thought it was just a stepwise undo, but it’s actually reverting the value to the Default value.

This is marked as a feature request because I guess it’s technically not broken, but I’d say it’s definitely extremely unexpected and unintuitive UI design for "keep as-is" to be indicated by the **absence** of the revert icon.

**Comment by dpolivaev** [comment]
Because all formatting properties not set by a level style come from the default style, I do not understand how you mean it. Please submit screenshots and example mind maps showing what you see and what you expect.

**Comment by euu2021** [comment]
> Any non-asserted values in styles should be clearly marked as such, i.e. as "unchanged" or "keep as-is" or whatever, instead of showing the values for the Default style.

This aspect of current the current UI is a result of [this discussion](https://sourceforge.net/p/freeplane/discussion/758437/thread/e6ff47e503/) from 2021. Previously, there was always a checkbox for each property (see a side by side comparison [at this post](https://sourceforge.net/p/freeplane/discussion/758437/thread/e6ff47e503/#feb2)). 

> Right now, when I create a new style from scratch, it’s pre-filled with the values taken from the "Default" style. However, they’re not actually "asserted", i.e. if I apply the newly created style to a node with automatic level styles, nothing changes. This is fairly unintuitive to me: If there’s a value visible in a style, I’d expect it to be applied.

I see what you mean. If the user sees an empty checkbox besides the property, he would more easily grasp that it's not an "asserted" property. 

> There’s also only a tiny visual difference when you change the default value from "keep as-is" to "overwrite": The tiny "Revert" button to the right. Which also confused me, because I thought it was just a stepwise undo, but it’s actually reverting the value to the Default value.

It seems that, at #1503 the same confusion happened.

So, those are the two principles in conflict here: more visually intuitive X less visual clutter.

Perhaps, a middle ground solution would be to bring back the checkboxes, but without the "Change" label besides each checkbox.
